src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  height: $card-height;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
```

Comment:
Avoid using fixed pixel values for width and height. Instead, consider using relative units or setting the width and height through a parent container to make the design more responsive. For example, you could use percentages or viewport units.